fixups/FileRedirectionFixup/GetPrivateProfileStringFixup.cpp

```diff
@@ -23,15 +23,21 @@ DWORD __stdcall GetPrivateProfileStringFixup(
             DWORD GetPrivateProfileStringInstance = ++g_FileIntceptInstance;
             if constexpr (psf::is_ansi<CharT>)
             {
-                LogString(GetPrivateProfileStringInstance,L"GetPrivateProfileStringFixup for fileName", widen(fileName, CP_ACP).c_str());
-                LogString(GetPrivateProfileStringInstance,L" Section", widen_argument(appName).c_str());
-                LogString(GetPrivateProfileStringInstance,L" Key", widen_argument(keyName).c_str());
+                if (fileName != NULL)
+                    LogString(GetPrivateProfileStringInstance,L"GetPrivateProfileStringFixup for fileName", widen(fileName, CP_ACP).c_str());
```

Comment:
Please include brackets, even on one line if and else blocks.

Reply:
Updated.

Reply:
done

Reply:
Thank you.  I don't see the change in your PR?  Did you push the changes?  For line 26-31

---
In reply to: [439763757](https://github.com/microsoft/MSIX-PackageSupportFramework/pull/145#discussion_r439763757) [](ancestors = 439763757)

Reply:
Sorry, I caught the ones below that and missed those 3.  Updated the PR for those lines.